### PR TITLE
AWS v11.5.1

### PR DESCRIPTION
**Pull request by njuettner**
Issue: https://github.com/giantswarm/giantswarm/issues/12584

### CODE REVIEW

File: aws/v11.5.1/release.diff [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    name: node-exporter                                                name: node-exporter
+    version: 1.2.0                                                     version: 1.2.0
+  components:                                                        components:
+  - name: app-operator                                               - name: app-operator
+    version: 1.0.0                                                     version: 1.0.0
+  - name: aws-cni                                                    - name: aws-cni
+    version: 1.6.0                                                     version: 1.6.0
+  - name: aws-operator                                               - name: aws-operator
+    releaseOperatorDeploy: true                                        releaseOperatorDeploy: true
+    version: 8.7.3                                              <      version: 8.7.5
```
Comment (xh3b4sd) [inline aws/v11.5.1/release.diff:47]: There are 2 versions but only one change. How does that work? 🙈

Comment (tfussell) [reply aws/v11.5.1/release.diff:47]: You mean the changelog doesn't show the changes from 8.7.4? I didn't get a chance to implement merging changelogs for intermediate releases yet, but I would like to do it soon.

Comment (xh3b4sd) [reply aws/v11.5.1/release.diff:47]: In this case I think we just skipped one version but I was wondering why and if we are not missing something in the changelogs. The diff is fine for me.

Comment (njuettner) [reply aws/v11.5.1/release.diff:47]: dependency for aws-operator (k8scloudconfig) with the correct calc for max pods was added in 8.7.5 😞 and not 8.7.4

Comment (xh3b4sd) [reply aws/v11.5.1/release.diff:47]: Still my question is not answered, what happened in the version before?

**Comment by xh3b4sd** [comment]
@giantswarm/team-firecracker let's get this out.

File: aws/v11.5.1/release.diff (hunk 1 above)
Comment (marians) [reply aws/v11.5.1/release.diff:47]: The aws-operator changelog shows:

- 8.7.5 changed: "Adjust number of host network pods on worker node for aws-cni"
- 8.7.4 fixed: "Adjust MAX_PODS for master and worker nodes to max IP's per ENI when using aws-cni"

I consider the two item as closely related and in my release notes curation work decided to summarize it all as

> Adjust `MAX_PODS` for master and worker nodes to max IPs per Elastic Network Interface (ENI)

If we lose important information here, please let me know.

**Pull request merged**